src/components/ChallengeCard/ChallengeCard.jsx

```diff
@@ -43,9 +44,9 @@ const ChallengeCard = ({ challenge }) => {
           <img src={Idea} alt="Idea Icon" />
           <div className="task-text">{challenge.task}</div>
         </div>
-        <div className="learn-more-button">
+        <Link to={"/challenge/" + challenge.id} className="learn-more-button">
```

Comment:
It's better written as `/challenge/${challenge.id}`

Reply:
I've updated this to use string templating